core/net/ip/resolv.h

```diff
@@ -43,6 +43,13 @@
 #include "contiki.h"
 #include "uip.h"
 
+/* If RESOLV_CONF_SUPPORTS_DNS_SD is set, then queries
+ * for services in the local TLD will use DNS-SD.
+ */
+#ifndef RESOLV_CONF_SUPPORTS_DNS_SD
+#define RESOLV_CONF_SUPPORTS_DNS_SD   (1)
```

Comment:
In resolve.c you wrote `#define RESOLV_CONF_SUPPORTS_DNS_SD 1`. Please be consistent.